python/graphstorm/gsf.py

```diff
@@ -656,4 +656,4 @@ def check_homo(g):
 
 def create_builtin_task_tracker(config):
     tracker_class = get_task_tracker_class(config.task_tracker)
-    return tracker_class(config)
+    return tracker_class(config.log_report_frequency)
```

Comment:
We are using config.log_report_frequency here but in other places and in the own-models.rst, we recommend users to use config.eval_frequency. Any specific reasons?

Reply:
Shall we use eval_frequency here and deprecate log_report_frequency

Reply:
Yes, agree to deprecate the log_report_frequency. This argument is confused. We's better to reduce the confusion.

I will remove the `log_report_frequency`, replacing it with eval_frequency.

Reply:
changed. And also remove the `log_report_frequency` from docs. So later we can deprecate this argument.